### ISSUE TITLE
Remote with ssh

### ISSUE THREAD

**Issue by sebtiz13** [GitFiend/Support#14]
Hello,
The remote with protocol ssh its not supported ?

I have just :
"Could not read from remote repository. Please make sure you have the correct access rights and the repository exists."

The support it's planned ?

**Comment by sebtiz13** [comment]
Yes i use ssh remote url

Command `git fetch` on Git Bash and others Git Client working

OS: Windows 10
My repo config:
`
[core]
	repositoryformatversion = 0
	filemode = false
	bare = false
	logallrefupdates = true
	symlinks = false
	ignorecase = true
[remote "origin"]
	url = ssh://git@mygitlab.fr:10022/myproject/myproject.git
	fetch = +refs/heads/*:refs/remotes/origin/*
[branch "master"]
	remote = origin
	merge = refs/heads/master
[branch "develop"]
	remote = origin
	merge = refs/heads/develop
`

**Comment by GitFiend** [comment]
That could be it. Gitfiend isn’t using git bash to run git commands. Does `git fetch` work in cmd? I think there’s an option when installing git for windows to add commands to cmd for git if not.  Unfortunately I don’t have access to a windows machine as I’m away on holiday right now.

**Comment by sebtiz13** [comment]
Yes `git fetch` work in cmd, but doesnt use ssh-agent, so the problem its maybe the suppot of ssh key ?
It's encrypted on AES-128 in CBC mode.

I dont know the probablies problems with key.

No problem it's not really urgent, I can use the commands for the moment. Good holiday
